packages/react-dom/src/__tests__/ReactDOMComponent-test.js

```diff
@@ -1394,6 +1394,42 @@ describe('ReactDOMComponent', () => {
         expect(console.log.calls.argsFor(1)[0]).toContain('onLoad called');
       }
     });
+
+    it('should work load event on <link> component', () => {
+      const container = document.createElement('div');
+      const onLoad = jest.fn();
+
+      ReactDOM.render(
+        <link href="http://example.org/link" onLoad={onLoad} />,
+        container,
+      );
+
+      const loadEvent = document.createEvent('Event');
+      const link = container.getElementsByTagName('link')[0];
+
+      loadEvent.initEvent('load', false, false);
+      link.dispatchEvent(loadEvent);
+
+      expect(onLoad).toHaveBeenCalledTimes(1);
+    });
+
+    it('should work error event on <link> component', () => {
```

Comment:
I am sorry for two final nits, but do you mind updating this to read:

```
it('should receive an error event on <link> elements')
```

Likewise for `load`. I'm also happy to make this change if you'd like.

Reply:
absolutely, thanks for the feedback